index.html

```diff
@@ -0,0 +1,363 @@
+            <br>
+            <p><strong>Built with: </strong>HTML, CSS, JavaScript</p>
+            <br>
+            <a href="https://github.com/morganheuer/guess-the-word" target="_blank"><p>Check out the code files here!</p></a>
+          </div>
+          <div class="card">
+            <a href="https://morganheuer.github.io/movie-search/" target="_blank">
+              <img src="assets/movie-search.png" alt="movie-search">
+            </a>
+            <h4>Movie Search</h4>
+            <p>Using concepts like functional components and hooks, I built this movie search that allows the user to type in a query, which initiates a call to an API (The Movie Database API) to retrieve information about a list of movies that are relevant to that search.
+            </p>
+            <br>
+            <p><strong>Built with: </strong>ReactJS, CSS Grid, API, Create React App</p>
+            <br>
+            <a href="https://github.com/morganheuer/movie-search/" target="_blank"><p>Check out the code files here!</p></a>
+          </div>
+          <!-- <div class="card">
+            <a href="" target="_blank">
+              <img src="/assets/ecsc.png" alt="books">
+            </a>
+            <h4>Emerald City Supper Club</h4>
+            <p>A group of friends and I came together to form the Emerald City Supper Club, a member-supported nonprofit with the mission of bringing friends and family together to raise awareness of, funds for, and a glass to organizations in our community that we’d like to support.</p>
+            <br>
+            <p>Eager to use my new WordPress custom theme building skills, I was able to reimagine and redesign our organization's website into something beautiful.</p>
+            <br>
+            <p><strong>Built with: </strong>HTML, CSS, PHP, WordPress Admin, WordPress Plugins, WordPress Themes, Figma</p>
+            <br>
+            <a href="https://github.com/morganheuer/EmeraldCitySupperClub/tree/main/wp-content/themes/ecsc21" target="_blank"><p>Check out the code files here!</p></a>
+          </div> -->
+          <!-- <div class="card">
+            <a href="http://morgandella.wpengine.com/" target="_blank">
+              <img src="/assets/book-nook.png" alt="books">
+            </a>
+            <h4>WP Book Nook</h4>
+            <p>Developed a custom WordPress theme to create a site to showcase all of the books I have been reading. By using the WordPress admin I am able to easily update my content when I finish a book.</p>
+            <br>
+            <p><strong>Built with: </strong>HTML, CSS, PHP, WordPress Admin, WordPress Plugins, WordPress Themes</p>
+          </div> -->
```

Comment:
Don't forget to delete commented out code! If you're wanting to save it for later, it's best to save it in a separate branch so it doesn't get delivered to your site visitors.